androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/surveydetail/SurveyDetailScreen.kt

```diff
@@ -1,32 +1,60 @@
 package co.nimblehq.avishek.phong.kmmic.android.ui.screen.surveydetail
 
+import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.pager.*
 import androidx.compose.material.*
 import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.tooling.preview.PreviewParameter
 import androidx.lifecycle.compose.collectAsStateWithLifecycle
 import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
 import co.nimblehq.avishek.phong.kmmic.presentation.module.HomeViewModel
+import co.nimblehq.avishek.phong.kmmic.presentation.uimodel.*
 import org.koin.androidx.compose.getViewModel
 
 @Composable
 fun SurveyDetailScreen(
     homeViewModel: HomeViewModel,
     surveyId: String,
-    onBackClick: () -> Unit
+    onBackClick: () -> Unit,
 ) {
     val viewState = homeViewModel.viewState.collectAsStateWithLifecycle()
     val surveyUiModel = viewState.value.surveys.find { it.id == surveyId }
+    var surveyQuestionUiModels by remember { mutableStateOf<List<SurveyQuestionUiModel>>(emptyList()) }
 
     surveyUiModel?.let {
-        SurveyStartContent(surveyUiModel = it,
+        SurveyStartContent(
+            surveyUiModel = it,
             onStartClick = {
                 // TODO: implement in the integrate task
+                // Call API and fetch the questions
+                surveyQuestionUiModels = List(5) {
+                    SurveyQuestionUiModel(
+                        id = it.plus(1).toString(),
+                        text = "How fulfilled did you feel during this WFH period?",
+                        displayType = DisplayType.DROPDOWN,
+                        imageUrl = surveyUiModel.largeImageUrl,
+                        answerUiModels = List(5) {
+                            AnswerUiModel(
+                                id = (it + 1).toString(),
+                                text = "Text ${it + 1}"
+                            )
+                        }
+                    )
+                }
             },
             onBackClick = onBackClick
```

Comment:
Cannot preview the screen
<img width="1440" alt="Screenshot 2023-05-22 at 12 35 09 PM" src="https://github.com/nimblehq/avishek-phong-kmm-ic/assets/32578035/90440179-4518-4fcb-9fe8-20a75b960a3b">

Reply:
Fixed in this commit https://github.com/nimblehq/avishek-phong-kmm-ic/pull/120/commits/f686f7addab4416fb526524b008aed5a74ee2412